docs/examples/driver_examples/Qcodes example with Rigol DS1074 Z.ipynb

```diff
@@ -0,0 +1,317 @@
+   "metadata": {},
+   "source": [
+    "## Importing dependencies "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import visa\n",
+    "\n",
+    "#Qcodes import\n",
+    "from qcodes.dataset.plotting import plot_dataset\n",
+    "from qcodes.instrument_drivers.rigol import DS1074Z\n",
+    "from qcodes.instrument.parameter import expand_setpoints_helper\n",
+    "from qcodes import initialise_database, load_or_create_experiment, Measurement"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create the instrument "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The address of the oscilloscope is needed to create the instrument. The address can also be found using the NI-MAX explorer.\n",
+    "\n",
+    "The instrument is created in the following way: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rigol = DS1074Z.RigolDrivers('rigol', 'USB0::0x1AB1::0x04CE::DS1ZB161650342::INSTR')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Connected to: RIGOL TECHNOLOGIES DS1074Z (serial:DS1ZB161650342, firmware:00.04.00) in 0.74s\n"
+     ]
+    }
+   ],
+   "source": [
+    "rigol.connect_message()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Trigger setup  "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Trigger source can be set to any channel (1 to 4). Here we use the input signal from channel 1 itself as a source for trigger. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rigol.trigger_edge_source('CHANnel1')\n",
+    "rigol.trigger_edge_slope('negative')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The trigger-mode type supported by this oscilloscope are `EDGe,PULSe, SLOPe, VIDeo, PATTern, DURATion`. Both the trigger mode and trigger level can be set in the following manner. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rigol.trigger_mode('PULS')\n",
+    "rigol.trigger_level(0.1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data acquisition and plotting"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The data acquisition is performed using measurement context manager. After initializing the database and loading/creating the experiment, the parameters are registered. The result are added to the database via datasaver object. The plot can be easily obtained using ``plot_by_id``.\n",
```

Comment:
this information with the general qcodes info does not need to be in the example notebook.
```suggestion
```

Reply:
ok.